src/Components/test/testassets/ComponentsApp.Server/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ComponentsApp.Server.Pages
+{
+    public class IndexModel : PageModel
```

Comment:
Do we need this? For clarity, can we drop it?

Reply:
I got rid of it. The item template creates it by default